### DISCUSSION TITLE
Difficulty Adjust Mod (Remove Blanket .5x Multiplier)

### DISCUSSION THREAD

**Discussion by Kevin2Holt** [ppy/osu#28710] [Ideas] [answered]
I feel like a blanket _"if-you-have-this-mod-on"_ approach doesn't fit with the Difficulty Adjust mod. Since you can either make it easier or harder. I like to play with a constant CS 5 which, in most cases (for the maps/star ratings I play), makes it harder (it shrinks the circles from their default).

Currently, if you make any changes using the Difficulty Adjust mod, you get a blanket .5x multiplier. I feel like the multiplier should be based more on the updated difficulty rather than having just a _"you-changed-something"_ multiplier.

### My Suggestion:
**Change the Difficulty Adjust multiplier to be based on the difference in star rating (between the no-mod default and the new difficulty-adjusted rating)**

The biggest question is whether this should allow for multipliers higher than 1x. I personally think it makes sense that if you make it harder, you get a higher multiplier. But, I also understand that the Difficulty Adjust mod is there primarily for practice (or messing around) and thus probably shouldn't make a higher score possible.

The other issue is when you adjust multiple attributes. When you adjust just one thing, like CS, it's pretty easy to see whether it's harder or easier. Once you add in a second or third (etc.) adjustment, it starts to be more difficult to judge how much more/less difficult it is, which is why I suggested having it based on the star rating difference (since it already calculates and displays the updated rating anyways).

**Comment by Stoppedpuma** [comment, marked as answer]
> I feel like a blanket "if-you-have-this-mod-on" approach doesn't fit with the Difficulty Adjust mod.

This isn't intended to be permanent. The problem in my eyes is adjusting the values to match mods such as HR, raising HP for example shouldn't contribute to multiplier whereas doing the opposite to match EZ should lower the multiplier. This means values should be defined for "what means what" which leads to the argument of balanced multipliers such as the HR should be worth more than HD debate or how some players find higher AR easier than lower AR which will also lead into low AR gimmick maps being affected.

There's plenty of discussions about DA being ranked or the score multiplier being adjusted. You can find some of them by just searching "DA" or "Difficulty adjust". There's also discussion about ranking the mod which can be found [here](https://github.com/ppy/osu/discussions/27799).

> Change the Difficulty Adjust multiplier to be based on the difference in star rating

This doesn't really work as I would say HR making a map CS 5.2 is harder than using DA to make it CS 5.3 since it also includes OD, HP, and AR changes yet DA gets a higher star rating.
